### ISSUE TITLE
Feature Request UI Frio - information regarding source of the post

### ISSUE THREAD

**Issue by utzer** [friendica/friendica#9258]
This is a feature request. 

On my computer I can see mouse over text for the red marked symbols. 
I would like to have some way to see this information on my mobile Android browser too. Is it possible to have this information available when tapping on the symbol? 

![image](https://user-images.githubusercontent.com/8361662/93860296-2ea8f800-fcbf-11ea-9143-0c82326bebac.png)

**Comment by MrPetovan** [comment]
We're using standard HTML attributes that I believe your mobile browser should support. I do not want to add a Javascript-based alternative just for mobile display.

**Comment by utzer** [comment]
> We're using standard HTML attributes that I believe your mobile browser should support. I do not want to add a Javascript-based alternative just for mobile display.

It does not work with Chrome, I can try some other browser if that helps. Maybe it is just the wrong HTML attribute?

**Comment by MrPetovan** [comment]
If it works on mouse over then it is the right HTML attribute. I'm not aware of another attribute for displaying tooltips without Javascript.

**Comment by utzer** [comment]
> If it works on mouse over then it is the right HTML attribute. I'm not aware of another attribute for displaying tooltips without Javascript.

Then let me rephrase the request. 

Can we add a menu to this little icons, similar to the menu you get when tapping on the Avatar, or any other solution without javascript it fine too. :-P

**Comment by MrPetovan** [comment]
It is possible but I personally don't want to be any part of that.

**Comment by utzer** [comment]
> It is possible but I personally don't want to be any part of that.

Lets do it like this, if there is possibility to have a tap on text or to have some other nice solution, then it would be great to implement something. If it really is not possible, I do not want cause JavaScript to be added with this feature request.

**Comment by Extarys** [comment]
Bootstrap have a tooltip component that Frio uses on some occasion, mainly on input fields. I believe all alt attributes should also use bootstrap tooltip for uniformity.

Doing so would also fix this issue on mobile.

Another possibility would be to add a information icon on the top right corner for mobile/tablet and on tap would open a small modal with the missing information.
